frontend/src/views/camp/Invitation.vue

```diff
@@ -1,66 +1,94 @@
 <template>
-  <auth-container>
-    <div v-if="!ready" class="text-center">
-      <v-progress-circular size="64" indeterminate color="primary" />
-    </div>
-    <div v-else-if="invitationFound === true">
-      <h1 class="display-1">
-        {{ $tc('views.camp.invitation.title', 0, { campName: invitation().campTitle }) }}
+  <div class="bg">
+    <div class="hill">
+      <v-icon :size="$vuetify.breakpoint.smAndUp ? 300 : 200" class="tent"
+        >$vuetify.icons.tentDay
+      </v-icon>
+      <img src="../../assets/invitation/tree-left.svg" alt="" class="tree l h-full" />
+      <img src="../../assets/invitation/tree-right.svg" alt="" class="tree r h-full" />
+      <div class="localnav justify-space-between d-flex col">
+        <ButtonBack v-if="!$vuetify.breakpoint.mdAndUp" text dark visible-label />
+        <UserMeta v-if="!$vuetify.breakpoint.mdAndUp" just-avatar />
```

Comment:
`UserMeta` is at the moment only used on Desktop and not on Mobile (the menu which opens doesn't exist in mobile so far). Is this introduced by purpose?

I can see that the menu makes sense. However, it's a bit inconsistent with the UI on `/camps`. There we also have an avatar on the right-top corner, however
- the avatar on `/camps` is smaller
- clicking doesn't open the menu but leads directly to the profile

Reply:
Implemented in camps